kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverterTest.java

```diff
@@ -686,56 +686,25 @@ public void testInvalidMapSchemalessNestedMapNullValue() {
     Assert.assertEquals(kafkaConnectMap, stringObjectMap);
   }
 
-  @Test
-  public void testInvalidMapSchemalessNullValue() {
```

Comment:
Why are we removing these tests? Looks like they're still present in WePay's fork: https://github.com/wepay/kafka-connect-bigquery/blob/4fb38a8f4b3b6db065976639c57106db9896f0db/kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverterTest.java#L658-L687

Reply:
No. It collided with the tests from upstream from last merge. We have duplicate tests in this file.

Reply:
Ahh, I see those now. Gotcha